web/src/main/java/com/epam/esm/security/filter/IsValidUser.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.esm.security.filter;
+
+import com.epam.esm.entity.Role;
+import com.epam.esm.security.configSecury.JwtAuthentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+@Component
```

Comment:
add space between imports and annotation

Reply:
fixed